### PR TITLE
IFF ammo factories

### PR DESCRIPTION
**Pull request by RenovatedAtol**
## About The Pull Request
Making all IFF ammo [T-25; T-29; T-81; TX8] having same factory recipe
Adding new factory recipe for T-29 drums and T-81 magazines
image

## Why It's Good For The Game
it's comfortable to have same factory ammo recipe for all smartguns

## Changelog
:cl:
add: T-29 drums factory
add: T-81 magazine factory
tweak: moved around tx8 ammo factory recipe
del: old tx8 ammo recipe
/:cl:

### CODE REVIEW

**Comment by private-tristan** [comment]
did you forget to change the "Your first factory phamplet" pamphlet that comes with the factory starterpack? I like looking at it because I don't play RO too often. it's "factoryhowto" in the code

File: code/modules/factory/outputter.dm [hunk 1]
```diff
@@ -159,20 +159,33 @@
 	refill_type = /obj/item/factory_part/smartgunner_rifle_box
 	refill_amount = 15
 
-/obj/item/factory_refill/mateba_speedloader_refill
+/obj/item/factory_refill/smartgunner_machinegun_magazine
```
Comment (BraveMole) [inline code/modules/factory/outputter.dm:162]: should be refill

File: code/modules/factory/outputter.dm [hunk 2]
```diff
@@ -159,20 +159,33 @@
...
 	name = "box of rounded metal plates"
 	desc = "A box with round metal plates inside. Used to refill Outputters."
-	refill_type = /obj/item/factory_part/mateba_speedloader
-	refill_amount = 30
+	refill_type = /obj/item/factory_part/smartgunner_machinegun_magazine
+	refill_amount = 12
 
-/obj/item/factory_refill/railgun_magazine_refill
+/obj/item/factory_refill/auto_sniper_magazine
```
Comment (BraveMole) [inline code/modules/factory/outputter.dm:168]: ditto

**Review by BraveMole — CHANGES_REQUESTED** [review]
Can you give me a cost comparison between buying these directly from req and making them in reqtorio? 

And do the same with current t25 ammo

**Review by BraveMole — APPROVED** [review]

**Pull request merged**